__tests__/app.test.js

```diff
@@ -13,7 +13,7 @@ afterAll(() => {
 });
 
 
-describe('app - working', () => {
+describe.only('app - working', () => {
```

Comment:
Remove this .only and push back up and if it all goes well then feel free to merge